src/entities/SkyBox.js

```diff
@@ -0,0 +1,32 @@
+class SkyBox extends Entity {
+    constructor() {
+        super();
+    }
+
+    /**
+     *
+     */
+    connected() {
+        // you can have texturesList be all individual textures
+        // or you can store them in a specified path and just
+        // load them up solely by filename in that path.
+        
+        this.texturesList = this.getAttribute('textures');
```

Comment:
@hanbollar what do we thing about using `src` instead of textures? We could also use the 'background' css property which would be sick.

Reply:
sounds good for src 👍  - i'll look into the background css property as well